src/psyclone/tests/psyir/backend/fortran_unsupported_declns_test.py

```diff
@@ -94,16 +94,16 @@ def test_fw_unsupported_interface_decln(tmpdir, fortran_writer):
     # Default visibility is public so there should be no explicit 'public ::'
     # statements.
     assert "public ::" not in code
-    # Add remaining structures so that we can generate compilable code
-    container.symbol_table.add(RoutineSymbol("eos1d"))
-    container.symbol_table.add(RoutineSymbol("eos2d"))
+    # Remove the my_sub RoutineSymbol before we readd it
+    container.symbol_table.remove(container.symbol_table.lookup("my_sub"))
```

Comment:
There is a typo in "readd", but before that, could we just not move the:
`container.addchild(Routine.create("my_sub", SymbolTable(), []))`
above in place of the 
`container.symbol_table.add(RoutineSymbol("my_sub", visibility=Symbol.Visibility.PUBLIC))`
so then we don't need to remove it?

Reply:
Yeah thats neater, done.